packages/api-kit/src/SafeApiKit.ts

```diff
@@ -258,6 +258,14 @@ class SafeApiKit {
     return sendRequest({
       url: `${this.#txServiceBaseUrl}/v1/safes/${address}/`,
       method: HttpMethod.Get
+    }).then((response: any) => {
+      // FIXME remove when service returns the singleton property
```

Comment:
```suggestion
      // FIXME remove when the service always returns the singleton property
```

I Think that this small change makes clear that the condition for removal is when the API always consistently returns the singleton property.